### PR TITLE
Initialised database with some sample data

### PR DESCRIPTION
**Pull request by Lunfeer**

### CODE REVIEW

**Review by MadeInShineA — CHANGES_REQUESTED** [review]
Sensor collection should be removed + Maintenances collection should be created + Some fields are missing inside the Plants collection

Also for future references, could you name your branche like this this-is-my-branch-name :)

File: server/main.js [hunk 1]
```diff
@@ -1,41 +1,140 @@
-import { Meteor } from 'meteor/meteor'
-import { LinksCollection } from '/imports/api/links'
+import { Meteor } from 'meteor/meteor';
+import { Random } from "meteor/random";
+import { PlantsCollection } from '/imports/api/plants';
+import { UsersCollection } from '/imports/api/users';
+import { ClimatesCollection } from '/imports/api/climates';
+import { SensorsCollection } from '/imports/api/sensors';
```
Comment (MadeInShineA) [inline server/main.js]: SensorsCollection should be removed

File: server/main.js [hunk 2]
```diff
@@ -1,41 +1,140 @@
...
+            _id: Random.id(),
+            name: "Arrosage",
+            frequency: 5,
+          },
+          {
+            _id: Random.id(),
+            name: "Sulfatage",
+            frequency: 150,
+          }
+        ]
```
Comment (MadeInShineA) [inline server/main.js]: Maintenances should have their own collection, therefore here, plants should only have a maintenance_id

File: server/main.js [hunk 3]
```diff
@@ -1,41 +1,140 @@
...
+      }),
+      PlantsCollection.insertAsync({ 
+        name: 'Basilic',
+        maintenances: [
+          {
+            _id: Random.id(),
+            name: "Arrosage",
+            frequency: 5,
+          },
+          {
```
Comment (MadeInShineA) [inline server/main.js]: Same as above

File: server/main.js [hunk 4]
```diff
@@ -1,41 +1,140 @@
...
+                    datasUrl: "url"
+                  }
+                ],
+                maintenancesDone: [
+                  {
+                    _id: Random.id(),
+                    maintenanceId: "id",
+                    date: new Date()
+                  }
+                ]
```
Comment (MadeInShineA) [inline server/main.js]: Same as above

File: server/main.js [hunk 5]
```diff
@@ -1,41 +1,140 @@
...
```
Comment (MadeInShineA) [inline server/main.js]: This should refer to an actual maintenance_id inside the maintenances collection

File: server/main.js [hunk 6]
```diff
@@ -1,41 +1,140 @@
...
+        temperatureRange: { min: 10, max: 35 }
+      })
+    ]);
+  }
 
-    await insertLink({
-      title: 'Discussions',
-      url: 'https://forums.meteor.com',
-    })
+  if ((await PlantsCollection.find().countAsync()) === 0) {
```
Comment (MadeInShineA) [inline server/main.js:25]: The plants should also include
• Expected growth duration 
• Harvest period

As required by the project document

**Comment by MadeInShineA** [comment]
You should also add a task field for the User

**Comment by MadeInShineA** [comment]
Could you also translate the place holder data into english please ? :)

**Review by MadeInShineA — APPROVED** [review]
Perfect thanks!

**Review by GDbateaux — APPROVED** [review]
Nice work

**Pull request merged**